README.md

```diff
@@ -7,8 +7,36 @@
 
 A PacMan clone written in C++ using SFML, with local multiplayer support.
 
+## Requirements for Building
+
+- C++17 (or newer) toolset.
+- Pre-built [SFML 2.6.x](https://www.sfml-dev.org/download/sfml/2.6.1/) or newer binaries. (Must match your Compiler and OS)
+- [CMake 3.20+](https://cmake.org/download/) or newer (add it to PATH)
+
+### For Windows
+
+- MS Visual Studio 2019 or newer (NOT vscode), with default "**Desktop C++ Development**" workload.
+- Please download "Visual C++ 64bit" edition of SFML; ignore others.
+- Move your unzipped `SFML-2.6.x` folder to its own home, example: `C:/SFML/SFML-2.6.1`
+- Make sure to edit [CmakeLists.txt](CMakeLists.txt) line 13, change value of `SMFL_HOME` to the directory you moved SFML into.
+
+### For MacOS
+
+- XCode latest (v13 or higher) from AppStore (with MacOS SDK)
+- Apple Developer tools. After Xcode is installed, run this in your Terminal:
+  ```bash
+  sudo xcode-select --install
+  ```
+- Please install SFML 2.6 on your Mac as shown in [official macOS guide](https://www.sfml-dev.org/tutorials/2.6/start-osx.php).
+
+### For Linux
+- Use your OS package manager to download SFML 2.5 or newer.
+- Alternatively, it's recommended to build SFML from source, see official docs.
+
 ## How it works
 
+- Please see [BUILDING.md](BUILDING.md) for build instructions.
+
```

Comment:
I don't think this should be in the "how it works" section.  Maybe at the end of the previous section ("requirements for building"?)